src/screens/LoginPage/LoginPage.tsx

```diff
@@ -189,6 +189,8 @@ function LoginPage(): JSX.Element {
         }
       } else {
         toast.warn('User not found!');
```

Comment:
@palisadoes  Would it be appropriate to update this to provide a more user-friendly message similar to what the comment above requests?

Reply:
Yes

Reply:
noted but I Think this particular toast is not for only the timeout session. It equally covers for a situation whereby the user isn't registered or unrecognised login. Perhaps  a wrong data was provided. This is where i created a different toast entirely for the logout session. Correct me if i am wrong though.

Reply:
Yeah, the condition does appear to cover a general issue with user data on login. That said, it may be more user friendly to add a more appropriate message if something is wrong. This may be out of scope for this ticket though.

Reply:
@Rumone should I just adjust this toast message within this PR or you want to create a different ticket for it?

Reply:
In this one